packages/terraform-modules-public/example/variables.tf

```diff
@@ -0,0 +1,240 @@
+# Provide the next vars with a vars-file or modifying the default value
+variable google {
+  description = "The GCP Data"
+  type        = map(string)
+
+  default = {
+    project = "my-project"
+    region  = "us-west1"
+    zone    = "us-west1-a"
+  }
+}
+
+variable network_name {
+  description = "The name of the new VPC network created"
+  type        = string
+
+  default = "celo-network"
+}
+
+variable celo_env {
+  description = "The celo network to connect with"
+  type        = string
+
+  default = "baklava"
+}
+
+variable replicas {
+  description = "The replica number for each component"
+  type        = map(number)
+
+  default = {
+    validator = 1 # Also used for proxy
+    txnode    = 0
+    attestation_service = 1
+  }
+}
+
+variable instance_types {
+  description = "The instance type for each component"
+  type        = map(string)
+
+  default = {
+    validator           = "n1-standard-1"
+    proxy               = "n1-standard-1"
+    txnode              = "n1-standard-1"
+    attestation_service = "n1-standard-1"
+  }
+}
+
+variable deploy_txnode_loadbalancer {
+  description = "Deploy the Load Balancer for Transmission nodes (if txnode > 0)"
+  type        = bool
+
+  default = false
+}
+
+variable proxy_accounts {
+  description = "The account data for the proxy nodes"
+  type        = map
+
+  default = {
+    account_addresses = [
+      "0xF2...",
+    ]
+    private_keys = [
+      "1b...",
+    ]
+    account_passwords = [
+      "secret1",
+    ]
+    private_node_keys = [
+      "1b...",
+    ]
+    enodes = [
+      "f8...",
+    ]
+  }
+}
+
+variable validator_accounts {
+  description = "The account data for the validator nodes"
+  type        = map
+
+  default = {
```

Comment:
Going off some questions from Discord from "reem" in #developerchat, we should make this clearer that it should be for `VALIDATOR_SIGNER_ADDRESS`, and also give instructions on how to get the private key for an account. The only way I know of that's semi-easy is what I suggested to him:

```
const Web3 = require('web3')
const w3 = new Web3()
const keystoreFileContents = <object found in the file corresponding to your address in your `<datadir>/keystore` directory>
console.log(w3.eth.accounts.decrypt(keystoreFileContents, yourAccountPassword)) // prints out an object with privateKey
```

Reply:
👋

Reply:
I updated some that part of the `README.md`. As I see this, if we/the user can provide all the needed parameters using only the `celocli account:new` command would be the ideal solution.
Maybe I am missing something but I think the user can create all the needed accounts with that command. So for example the user should create the accounts for the validators (these ones not included in the terraform variables), proxy (only for the enode address), validator signers and attestation service, and the last two has to be authorized using the celocli.

If so the used should not care about decrypting a keystore json file to a private key.

Also I find a bit difficult if an user want to deploy 10, 20 or 30 validators stacks (validator+proxy+signer+attestation service), and managing 4*replicas mnemonics maybe is not the better solution. 
I would say it would be nice include a section about managing accounts (with celotool or with some tool like `ethereum_hdwallet`), but I am not sure if that section would fit better in the general documentation rather than the terraform module. I have some scripts